### PR TITLE
Update data.json

### PR DESCRIPTION
**Pull request by stopWarByWar**
add bas to data.json

### CODE REVIEW

**Review by Jeffery-ldj — DISMISSED** [review]
It seems that you don't support Greenfield on your website.

**Comment by stopWarByWar** [comment]
> It seems that you don't support Greenfield on your website.

User can create off chain attestation and store it into greenfield on BASCAN. Here is example:
https://bascan.io/attestation/create?schema=0xdbf421e0db21a22185e56d57fda82dd48e8d863a65335d665c992ac31b141139

File: data.json
Comment (CharlieInRain) [inline data.json:1]: Could you please resolve the conflict with main branch?

**Comment by Jeffery-ldj** [comment]
You can close this PR now. We already added your tool.

**Pull request closed without merging**